code/modules/surgery/organs/ears.dm

```diff
@@ -93,4 +93,25 @@
 	..()
 	color = H.hair_color
 	H.dna.features["ears"] = "None"
-	H.update_body()
\ No newline at end of file
+	H.update_body()
+
+/obj/item/organ/ears/robot
+	name = "robotic ears"
+	icon_state = null
+	origin_tech = "engineering=2"
+	desc = "A pair of microphones, installed in the head, that grant the ability to hear."
+	zone = "head"
+	slot = "ears"
+	gender = PLURAL
+
+/obj/item/organ/ears/robot/emp_act(severity)
+	switch(severity)
+		if(1)
+			owner.Jitter(120)
+			owner.Dizzy(120)
+			deaf = 1
```

Comment:
`deaf` ticks down every two seconds. Make it a higher value to make it meaningful. Maybe somewhere around 20 seconds?